handler/src/test/java/io/netty/handler/ssl/SSLEngineTest.java

```diff
@@ -3643,12 +3671,24 @@ private void testSessionAfterHandshake0(
             assertEquals(Boolean.TRUE, clientEngine.getSession().getValue(key));
             serverEngine.getSession().putValue(key, Boolean.TRUE);
             assertEquals(Boolean.TRUE, serverEngine.getSession().getValue(key));
+            clientEngine.getSession().removeValue(key);
+            serverEngine.getSession().removeValue(key);
```

Comment:
Did you intend to have these removes here? It makes the `assertNull(clientSession.getValue(key));` check below, and the `assertEquals(0, engine.getHandshakeSession().getValueNames().length);` check in the trust manager, of questionable value.